std/experimental/logger/core.d

```diff
@@ -1631,7 +1631,7 @@ private @property Logger defaultSharedLoggerImpl() @trusted
     import std.conv : emplace;
     import std.stdio : stderr;
 
-    static __gshared ubyte[__traits(classInstanceSize, FileLogger)] _buffer;
+    static __gshared align(FileLogger.alignof) void[__traits(classInstanceSize, FileLogger)] _buffer;
```

Comment:
@ZombineDev the singleton wouldn't appear at module level - that's a global. It needs to be a function returning a reference.